packages/core/admin/admin/src/content-manager/hooks/useLazyComponents/index.js

```diff
@@ -11,13 +11,16 @@ const componentStore = new Map();
  */
 const useLazyComponents = (componentUids = []) => {
   const [lazyComponentStore, setLazyComponentStore] = useState(Object.fromEntries(componentStore));
-  const [loading, setLoading] = useState(() => {
-    if (componentStore.size === 0 && componentUids.length > 0) {
-      return true;
-    }
+  /**
+   * Start loading only if there are any components passed in
+   * and there are some new to load
+   */
+  const newUids = useMemo(
+    () => componentUids.filter((uid) => !componentStore.get(uid)),
+    [componentUids]
+  );
+  const [loading, setLoading] = useState(() => !!newUids.length);
```

Comment:
Seems like its easier just to "start" loading only if there is new uids to add to store.

Reply:
I'll add tests to check out this issue